### ISSUE TITLE
[🔌 Provider]: Http Provider description

### ISSUE THREAD

**Issue by Matvey-Kuk** [keephq/keep#1666]
Update: Add a hint to the HTTP provider's page to go to Settings->Webhook if user is looking for the inbound webhook.

Old:

For now, http provider is used only for outbound requests. I caught myself trying to use it to send alerts to Keep. 
<img width="1465" alt="Screenshot 2024-08-20 at 12 58 10" src="https://github.com/user-attachments/assets/d51a4bb8-2064-4e1b-9f8c-2fa69c45ee6b">
Worth updating the provider with ability to ingest (free-form?) payloads as alerts + add instruction and dynamic URL + Auth example using Curl. Similar to Prometheus provider.

**Comment by talboren** [comment]
Is it different from generic webhook?

**Comment by shahargl** [comment]
How is it different from this:
![CleanShot 2024-08-20 at 13 07 18@2x](https://github.com/user-attachments/assets/089fd487-7499-45c5-b546-fdb55e0e015d)

**Comment by shahargl** [comment]
I see it as a  "build your own provider" - a way to define your own integration. e.g. a form that you upload a logo, name and some extraction rules, click "connect" and get a URL e.g. `/alerts/event/custom_providers/{name}` that you can start sending alerts to and see it nicely on the UI

**Comment by Matvey-Kuk** [comment]
Good catch. What I meant is not different from what @talboren mentioned. I would be happy even if we just help users to find webhooks from the Http provider page. Smth like:

"If you are looking for a webhook to shoot alerts to Keep, please go to..."

**Comment by Matvey-Kuk** [comment]
Updating the description.

**Comment by talboren** [comment]
https://github.com/keephq/keep/issues/466

**Comment by Matvey-Kuk** [comment]
HTTP provider is removed from the list of providers
